src/renderer/components/kube-object/kube-object-details.tsx

```diff
@@ -169,6 +166,18 @@ export class KubeObjectDetails extends React.Component {
         <item.components.Details object={object} key={`object-details-${index}`} />
       ));
 
+    if (details.length === 0) {
+      const crd = crdStore.getByObject(object);
+
+      /**
+       * This is a fallback so that if a custom resource object doesn't have
+       * any defined details we should try and display at least some details
+       */
+      if (crd) {
+        details.push(<CrdResourceDetails key={object.getId()} object={object} crd={crd} />);
+      }
```

Comment:
By fixing the typing on `CrdResourceDetails` (to not saying we want a `CustomResourceDefenition` as `object`, because we don't) it is no longer necessary to use a type cast.